src/Codeception/Module/Doctrine2.php

```diff
@@ -411,6 +418,141 @@ public function haveInRepository($entity, array $data)
         }
     }
 
+    /**
+     * Loads fixtures. Fixture can be specified as a fully qualified class name,
+     * an instance, or an array of class names/instances.
+     *
+     * ```php
+     * <?php
+     * $I->haveFixtures(AppFixtures::class);
+     * $I->haveFixtures([AppFixtures1::class, AppFixtures2::class]);
+     * $I->haveFixtures(new AppFixtures);
+     * ```
+     *
+     * By default fixtures are loaded in 'append' mode. To replace all
+     * data in database, use `false` as second parameter:
+     *
+     * ```php
+     * <?php
+     * $I->haveFixtures(AppFixtures::class, false);
+     * ```
+     *
+     * Note: this method requires `doctrine/data-fixtures` package to be installed.
+     *
+     * @param string|string[]|object[] $fixtures
+     * @param bool $append
+     * @throws ModuleException
+     * @throws ModuleRequireException
+     */
+    public function haveFixtures($fixtures, $append = true)
```

Comment:
It would be better to rename this method to `loadFixtures`, right?

Reply:
Agree. Fixed.